tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1384,6 +1387,8 @@ Status StorageManager::get_fragment_info(
   if (!sparse)
     array_schema->domain()->expand_to_tiles(&expanded_non_empty_domain);
 
+  meta.load_rtree(encryption_key);
```

Comment:
Same as above, this needs to be called outside the `get_fragment_info` function, whenever the new API is called.